pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2784,6 +2784,20 @@ self: super: {
   # https://github.com/brandonchinn178/tasty-autocollect/issues/54
   tasty-autocollect = dontCheck super.tasty-autocollect;
 
+  # Megaparsec 9.7. merged, but unreleased
+  configurator-pg = doJailbreak super.configurator-pg;
+
+  # Remove one upper bounds for tasty-quickcheck are bumped.
```

Comment:
There is a section in configuration-common where a lot of these tasty-quickcheck overrides are grouped together. Though, maybe not worth it moving these around since this can be dropped on the next hackage bump IIUIC?